test/plots/areaPlotTests.ts

```diff
@@ -2,162 +2,177 @@
 
 describe("Plots", () => {
   describe("AreaPlot", () => {
-    // HACKHACK #1798: beforeEach being used below
     it("renders correctly with no data", () => {
-      let svg = TestMethods.generateSVG(400, 400);
+      let SVG_WIDTH = 400;
+      let SVG_HEIGHT = 500;
+      let svg = TestMethods.generateSVG(SVG_WIDTH, SVG_HEIGHT);
       let xScale = new Plottable.Scales.Linear();
       let yScale = new Plottable.Scales.Linear();
       let plot = new Plottable.Plots.Area();
-      plot.x((d: any) => d.x, xScale);
+      plot.x((d) => d.x, xScale);
       plot.y((d) => d.y, yScale);
       assert.doesNotThrow(() => plot.renderTo(svg), Error);
-      assert.strictEqual(plot.width(), 400, "was allocated width");
-      assert.strictEqual(plot.height(), 400, "was allocated height");
+      assert.strictEqual(plot.width(), SVG_WIDTH, "was allocated width");
+      assert.strictEqual(plot.height(), SVG_HEIGHT, "was allocated height");
```

Comment:
Unsure if the above two tests really are useful.  I would remove them; your call.